src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {t, tct} from '../../locale';
+
+import OrganizationState from '../../mixins/organizationState';
+import Confirmation from './confirmation';
+
+const TodoItem = React.createClass({
+  propTypes: {
+    task: PropTypes.object,
+    onSkip: PropTypes.func.isRequired
+  },
+
+  mixins: [OrganizationState],
+
+  getInitialState: function() {
+    return {
+      showConfirmation: false,
+      isExpanded: false
+    };
+  },
+  toggleDescription(e) {
+    this.setState({isExpanded: !this.state.isExpanded});
+  },
+
+  toggleConfirmation: function() {
+    this.setState({showConfirmation: !this.state.showConfirmation});
+  },
+
+  formatDescription: function() {
+    if (this.state.isExpanded) {
+      return <p>{this.props.task.description}. {this.props.task.detailedDescription}</p>;
```

Comment:
nit, but maybe combine these into `return <p>{this.props.task.description} {this.state.isExpanded && ('. ' +this.props.task.detailedDescription)}</p>;`